src/component/modal/EditModal.jsx

```diff
@@ -1,13 +1,12 @@
-import React, {useEffect, useState} from 'react'
+import React, { useEffect, useState } from "react";
 import Box from "@mui/material/Box";
 import Modal from "@mui/material/Modal";
-import Button from "@mui/material/Button";
 import Typography from "@mui/material/Typography";
 import EditOutlinedIcon from "@mui/icons-material/EditOutlined";
 import TextareaAutosize from "@mui/material/TextareaAutosize";
 import { height } from "@mui/system";
 import { useDispatch, useSelector } from "react-redux";
-import { fetchPost } from '../../redux/slices/postSlice';
+import { fetchPost } from "../../redux/slices/postSlice";
```

Comment:
I would suggest you to use absolute paths and noth '../../'. It was giving an error while i was deploying it to Netlify then I had to change the imports